File: beginner/pagina08/2547/2547.py
while True:
    try:

        participantes, altura_minima, altura_maxima = [
            int(x) for x in input().split()
        ]

        total_participantes_habeis = 0

        for _ in range(participantes):
            altura = int(input())
            if altura_minima <= altura <= altura_maxima:
                total_participantes_habeis += 1

        print(total_participantes_habeis)

    except EOFError:
        braak

File: beginner/pagina08/2547/2547.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("\n")

let caso;
while (caso = input.shift()) {

    const [participantes, altura_minima, altura_maxima] = caso.split(" ").map(Number)

    let total_participantes_habeis = 0
    for(let i = 0; i < participantes; i++){
        const altura = Number(input.shift())
        if ( altura <= altura_maxima && altura >= altura_minima){
            total_participantes_habeis++
        }
    }

    console.log(total_participantes_habeis)
}